xml/decoders/esu/v5lpOutputLabels.xml

```diff
@@ -9,90 +9,106 @@
 
     <output name="1,36" label="|"/> <!-- suppress reserved item -->
 
-    <output name="3,1" label="Optional Load"/>
+    <output name="3,1" label="Optional Load">
+        <label xml:lang="de">Alternative Last</label>
+	</output>
     <output name="3,2" label="Shunting Mode">
         <label xml:lang="ca">Mode de Maniobra</label>
         <label xml:lang="cs">Režim posunu</label>
         <label xml:lang="da">Shunting Mode</label>
+        <label xml:lang="de">Rangiergang</label>
         <label xml:lang="nl">Shunting Mode</label>
     </output>
     <output name="3,3" label="Brake 1">
         <label xml:lang="ca">Fre 1</label>
         <label xml:lang="cs">Brzda 1</label>
         <label xml:lang="da">Brake 1</label>
+        <label xml:lang="de">Bremsen 1</label>
         <label xml:lang="nl">Brake 1</label>
     </output>
     <output name="3,4" label="Brake 2">
         <label xml:lang="ca">Fre 2</label>
         <label xml:lang="cs">Brzda 2</label>
         <label xml:lang="da">Brake 2</label>
+		<label xml:lang="de">Bremsen 2</label>
```

Comment:
Do you use a text editor that can convert TABs to spaces? The reason the indentation is wrong is that you use TABs instead of spaces. It's strongly recommended to use spaces instead of TABs since different users have different TAB sizes.

Reply:
Yes, that could be the case. I use Notepad++ to edit the xml files and the indentation is mostly automatic. I will make sure not to set tabs in the future.